File: 217 Contains Duplicate/contains_duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        if (nums.Length < 2) return false;
        HashSet<int> set = new HashSet<int>();
        foreach (var n in nums)
        {
            if (set.Contains(n)) return true;
            set.Add(n);
        }
        return false;
    }
} // Author: XC

File: 217 Contains Duplicate/contains_duplicate.java
public class Solution {
    public boolean containsDuplicate(int[] nums) {
        if (nums.length < 2) return false;
        HashSet<Integer> set = new HashSet<Integer>();
        for (int n : nums) {
            if (set.contains(n)) return true;
            set.add(n);
        }
        return false;
    }
} // Author: XC
